### ISSUE TITLE
Training detail on WebQuestions and CuratedTREC

### ISSUE THREAD

**Issue by yeliu918** [facebookresearch/DPR#142]
Hi,

I wonder how many epochs do you train on small dataset WebQuestions and CuratedTREC? And Could you provide some details like batch_size, learning rate for them?

Best,

**Comment by vlad-karpukhin** [comment]
Hi, 
The parameters are basically the same except num of train epochs, I used 200 with the best checkpoint selected somewhere at 150-th epoch.

**Comment by vlad-karpukhin** [comment]
The datasets have been added
See https://github.com/facebookresearch/DPR/issues/143
